src/keyboard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component, ReactNode } from 'react';
+
+type State = {
+  key: string,
```

Comment:
"Make the `App` a class component with `pressedKey` in the `state`." -requirement  from readme.md file